submissions/kazeemErinfolami/javascript/Dom/footer.js

```diff
@@ -0,0 +1,41 @@
+let newsSubmit = document.getElementById("newsLetter"),
+  newsEmail = document.getElementById("newsEmail");
+
+//required Error message
+let newsEmailRequired = "⚠️Please enter your email";
+//required validation Error message
+let newsEmailError = "⚠️Please enter a valid email";
+
+//validation
+function newsValidation(
```

Comment:
Since you are just using this function for the email field. kindly pass the email direct and also all the error messages. No need for the parameters being pass into this function except for the `event` parameter